src/coreclr/src/interop/comwrappers.hpp

```diff
@@ -17,8 +17,9 @@ enum class CreateComInterfaceFlagsEx : int32_t
 
     // Highest bit is reserved for internal usage
     IsPegged = 1 << 31,
+    IsComActivated = 2 << 31,
```

Comment:
This should really be above `IsPegged` and be `IsComActivated = 1 << 30`

Reply:
🤦‍♀